portal/templates/scheduled_jobs_list.html

```diff
@@ -10,20 +10,25 @@ <h4>{{ _("Current Jobs") }}</h4>
     <ul>
         {% for job in jobs %}
             <li>
-                <button title="Run Job" onclick="triggerJob(event, {{ job.id }})"><i class="fa fa-play"></i></button>
-                <b>{{ job.name }}</b>
-                <br />{{ _("Task") }}: {{ job.task }}
-                {% if job.args %}
-                <br />{{ _("Args") }}: {{ job.args }}
-                {% endif %}
-                {% if job.kwargs %}
-                <br />{{ _("Kwargs") }}: {{ job.kwargs }}
-                {% endif %}
-                <br />{{ _("Schedule") }}: {{ job.schedule }}
-                <br />{{ _("Active") }}: {{ job.active }}
-                <br />{{ _("Last Runtime") }}: {{ job.last_runtime }}
-                <br />{{ _("Last Status") }}: {{ job.last_status }}
-                <div id="triggerMessage_{{job.id}}" class="text-info"></div>
+                <div id="job_{{ job.id }}">
+                    <button title="Toggle Job" onclick="toggleJob(event, {{ job.id }})"><i id="activeIcon_{{job.id}}" class="{{'fa fa-toggle-on' if job.active else 'fa fa-toggle-off'}}" /></i></button>
+                    <button title="Run Job" onclick="triggerJob(event, {{ job.id }})"><i class="fa fa-play"></i></button>
+                    <b>{{ job.name }}</b>
+                    <div id="activeText_{{job.id}}" class="{{'text-info' if job.active else 'text-danger'}}">{{ "Active" if job.active else "Inactive" }}</div>
+                    {{ _("Task") }}: {{ job.task }}
```

Comment:
I don't think we'll want to translate this content (notes that would only help a developer), at least for now. Will lead to a lot of questions from translators :).

There isn't any way to disable extraction, just from this file, is there? There's always the possibility they may change their minds...

Reply:
Well, right now it's just a few field labels getting translated. But I can just change those to straight strings (aka `Task` instead of `{{ _("Task") }}`) if we don't want those translated, sure.

Reply:
Thanks! The only ones I think they might balk at are `Kwargs` and `Args`